src/main/scala/dpla/ingestion3/mappers/providers/RumseyMapping.scala

```diff
@@ -0,0 +1,154 @@
+      .map(stringOnlyTimeSpan)
+
+  override def description(data: Document[NodeSeq]): Seq[String] =
+    extractStrings(metadataRoot(data) \ "description")
+
+  override def extent(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(metadataRoot(data) \ "extent")
+
+  override def format(data: Document[NodeSeq]): Seq[String] =
+    extractStrings(metadataRoot(data) \ "format")
+      .flatMap(_.splitAtDelimiter(";"))
+      .filterNot(isDcmiType)
+
+  override def identifier(data: Document[NodeSeq]): Seq[String] =
+    extractStrings(metadataRoot(data) \ "identifier")
+
+  override def language(data: Document[NodeSeq]): Seq[SkosConcept] =
+    extractStrings(metadataRoot(data) \ "language")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyConcept)
+
+  override def place(data: Document[NodeSeq]): Seq[DplaPlace] =
+    extractStrings(metadataRoot(data) \ "coverage")
+      .flatMap(_.splitAtDelimiter("--"))
+      .map(nameOnlyPlace)
+
+  override def publisher(data: Document[NodeSeq]): Seq[EdmAgent] =
+    extractStrings(metadataRoot(data) \ "publisher")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyAgent)
+
+  override def relation(data: Document[NodeSeq]): Seq[LiteralOrUri] =
+    extractStrings(metadataRoot(data) \ "relation")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(eitherStringOrUri)
+
+  override def replacedBy(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(metadataRoot(data) \ "isReplacedBy")
+
+  override def replaces(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(metadataRoot(data) \ "replaces")
+
+  override def rightsHolder(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    extractStrings(metadataRoot(data) \ "rightsHolder")
+      .map(nameOnlyAgent)
+
+  // done, sbw. hard-coded
+  override def rights(data: Document[NodeSeq]): Seq[String] =
+    Seq("Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported")
+
+  override def subject(data: Document[NodeSeq]): Seq[SkosConcept] =
+    extractStrings(metadataRoot(data) \ "subject")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyConcept)
+
+  override def temporal(data: Document[NodeSeq]): ZeroToMany[EdmTimeSpan] =
+    extractStrings(metadataRoot(data) \ "temporal")
+      .map(stringOnlyTimeSpan)
+
+  override def title(data: Document[NodeSeq]): Seq[String] =
+    extractStrings(metadataRoot(data) \ "title")
+
+  //  hardcoded to `image`
+  override def `type`(data: Document[NodeSeq]): Seq[String] = Seq("image")
+
+  // OreAggregation
+  override def dplaUri(data: Document[NodeSeq]): ZeroToOne[URI] = mintDplaItemUri(data)
+
+  // done, sbw hard-coded
+  override def dataProvider(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    Seq(EdmAgent(name = Some("David Rumsey")))
+
+  // done, sbw hard-coded
+  override def edmRights(data: Document[NodeSeq]): ZeroToMany[URI] =
+    Seq(URI("https://creativecommons.org/licenses/by-nc-sa/3.0/"))
```

Comment:
We should probably set up constants for these URIs. The canonical URI is `http`, not `https`

Reply:
@mdellabitta Did I/we muck the edmRights normalization? 

https://github.com/dpla/ingestion3/blob/5eb531d24586ab7bbeb7239a93b2ec903fd3d39e/src/main/scala/dpla/ingestion3/enrichments/normalizations/StringNormalizations.scala#L76-L85

Reply:
Looks like it: https://rightsstatements.org/page/1.0/?language=en#collection-nc

D'oh!